src/bridge.cpp

```diff
@@ -125,7 +125,7 @@ create_bidirectional_bridge(
   const std::string & topic_name,
   size_t queue_size)
 {
-  RCLCPP_INFO(ros2_node->get_logger(), "create bidirectional bridge for topic " + topic_name);
+  RCLCPP_INFO(ros2_node->get_logger(), ("create bidirectional bridge for topic " + topic_name).c_str());
```

Comment:
Sorry, but now I remember that passing a "non-literal" string is a security issue, and that's why support for `std::string` as a format string was removed.

The strictly correct code would be:

```suggestion
  RCLCPP_INFO(ros2_node->get_logger(), "create bidirectional bridge for topic "%s", topic_name.c_str());
```

but I don't mind if we merge this as-is first.

Reply:
I think there is a stray quote in yours, so I went with: 

```
  RCLCPP_INFO(ros2_node->get_logger(), "create bidirectional bridge for topic %s", topic_name.c_str());
```